__tests__/app.test.js

```diff
@@ -127,3 +127,61 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  it("should return requested comments", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((response) => {
+        const comments = response.body.comments;
+        if (comments.length > 0) {
```

Comment:
if comments.length is 0, this test will still pass - we should check that the length is what we expect it to be first